RoddyCore/src/de/dkfz/roddy/config/loader/ConfigurationFactory.groovy

```diff
@@ -248,10 +259,16 @@ class ConfigurationFactory {
      * @see PreloadedConfiguration
      *
      * @param file The config file.
-     * @return An object containing basic information about a configuration.
+     * @return An object containing basic information about a configuration OR null, if the no preloaded config could be loaded.
      */
     PreloadedConfiguration loadInformationalConfigurationContent(File file) {
         String text = loadAndPreprocessTextFromFile(file)
+
+        if (!text) {
+            logger.rare("Could not identify file ${file.absolutePath} as a Roddy configuration file.")
```

Comment:
Also here, if possible, add the action that Roddy takes? Silently ignore (skip)?

Reply:
Fixed.